ee/get-support.md

```diff
@@ -45,6 +45,26 @@ This support dump only contains logs for the node where you're running the
 command. If your UCP is highly available, you should collect support dumps
 from all of the manager nodes.
 
+## Use the CLI to get a support dump (multiple nodes)
+
+To get the support dump from the CLI for multiple nodes, use SSH to log into a UCP manager node
+and run:
+
+To generate support dump details for all hosts in a cluster, from one remote host, please see below.
```

Comment:
This is a short doc, so you can remove this sentence.

Reply:
completed.